Can't build upas/nfs

```
keks@keks-x220 ~/src/plan9port/src/cmd/upas/nfs$ mk install
9c  box.c
9c  decode.c
decode.c:146:8: warning: variable ‘argv’ set but not used [-Wunused-but-set-variable]
  char *argv[4];
        ^~~~
9c  fs.c
fs.c:953:47: warning: variable ‘reset’ set but not used [-Wunused-but-set-variable]
  int i, j, c, type, flag, unflag, flagset, f, reset;
                                               ^~~~~
9c  imap.c
imap.c:348:6: warning: variable ‘prefix’ set but not used [-Wunused-but-set-variable]
  int prefix, len;
      ^~~~~~
9c  main.c
9c  sx.c
9c  thread.c
9c  util.c
9l -o o.mailfs box.o decode.o fs.o imap.o main.o sx.o thread.o util.o 
(.text+0x23): undefined reference to `pthread_detach'
(.text+0xa9): undefined reference to `pthread_mutex_trylock'
(.text+0xdd): undefined reference to `pthread_mutexattr_init'
(.text+0xe7): undefined reference to `pthread_mutexattr_settype'
(.text+0xfa): undefined reference to `pthread_mutexattr_destroy'
(.text+0x23b): undefined reference to `pthread_create'
(.text+0x28c): undefined reference to `pthread_getspecific'
(.text+0x29f): undefined reference to `pthread_setspecific'
(.text+0x2f3): undefined reference to `pthread_key_create'
(.text+0x339): undefined reference to `pthread_key_create'
collect2: error: ld returned 1 exit status
mk: 9l -o o.mailfs ...  : exit status=exit(1)
```

--- a/bin/9l
+++ b/bin/9l
@@ -40,7 +40,7 @@ case "$tag" in
 	userpath=true
 	extralibs="$extralibs -lutil -lresolv"
 	case "${SYSVERSION:-`uname -r`}" in
-	2.6.* | 3.*)
+	2.6.* | 3.* | 4.*)
 		extralibs="$extralibs -lpthread"
 		;;
 	esac

